pkg/products/apicurito/reconciler.go

```diff
@@ -185,8 +185,10 @@ func (r *Reconciler) reconcileComponents(ctx context.Context, installation *inte
 
 	or, err := controllerutil.CreateOrUpdate(ctx, serverClient, apicuritoCR, func() error {
 		// Specify 2 pods to provide HA
-		apicuritoCR.Spec.Size = 2
-		apicuritoCR.Spec.Image = "registry.redhat.io/fuse7/fuse-apicurito:1.5"
+		apicuritoCR.Spec.Size = 3
+		//apicuritoCR.Spec.Image = "registry.redhat.io/fuse7/fuse-apicurito:1.5"
```

Comment:
Is there a reason for keeping this commented out line? :thinking:

Reply:
@KevFan no, also, I think we can drop Size back to 2. Will update